python/python/glide/async_commands/server_modules/json.py

```diff
@@ -376,7 +431,7 @@ async def type(
             Defaults to None.
 
     Returns:
-        Optional[Union[bytes, List[bytes]]]:
+        Optional[TJsonUniversalResponse[bytes]]:
```

Comment:
Isn't just a `List[TEncodable]` (I follow given examples)?
If it could be `None` - show an example (and test) with `None`. If it could be a string/bytes - show it as well. Given examples show list of b strings only.

Reply:
Could be a single byte (for not json path) or list of bytes for json path 

If key / path doesn't exist - None

So it optional (union (T, list of T))